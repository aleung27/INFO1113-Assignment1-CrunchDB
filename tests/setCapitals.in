SET a 4
SET A 0
GET a
GET A
SET AA 7
SET aa 1
LIST ENTRIES
