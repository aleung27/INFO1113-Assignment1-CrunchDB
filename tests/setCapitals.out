> ok

> ok

> [4]

> [0]

> ok

> ok

> aa [1]
AA [7]
A [0]
a [4]

